### DISCUSSION TITLE
How to use the built-in web server?

### DISCUSSION THREAD

**Discussion by mattl** [datenstrom/community#281] [Ask a question] [answered]
I downloaded yellow, unzipped it and ran ``php -S localhost:3000``

I got this in my browser.

```
Not Found
The requested resource / was not found on this server.
```

**Comment by schulle4u** [comment, marked as answer]
you have to run the built-in webserver as follows: 

`php -S localhost:3000 yellow.php`

Just tested under Windows 10/Linux Subsystem with latest Yellow. 

See also datenstrom/community#237

HTH
Steffen

**Comment by verbertus** [comment]
Hi, my first post here. Just discovered Yellow yesterday. Read all the docs and had no headache afterwards! 
First thing I did was install XAMPP on my computer, to have a local server. 
Now I find this thread about a server built-in! Great!
So I do not need XAMPP to run Yellow locally?
Thanks!

**Comment by schulle4u** [comment]
Yes, for testing purposes you can use php's [built-in webserver](https://php.net/manual/en/features.commandline.webserver.php) to run Yellow locally.
